content/03.categorize.md

```diff
@@ -446,15 +446,13 @@ can be used before overfitting occurs. For example, the unsupervised and
 semi-supervised methods that we have discussed reduce the need for labeled
 examples [@doi:10.1016/j.jbi.2016.10.007]. The anchor and learn framework
 [@doi:10.1093/jamia/ocw011] uses expert knowledge to identify high-confidence
-observations from which labels can be inferred. The adversarial training example
-strategies mentioned above can reduce overfitting, if transformations are
-available that preserve the meaningful content of the data while transforming
-irrelevant features [@doi:10.1101/095786]. While adversarial training examples
+observations from which labels can be inferred.
+The strategies of adversarial training mentioned above can reduce overfitting, if transformations are available that preserve the meaningful content of the data while transforming irrelevant features [@doi:10.1101/095786].
+While adversarial training examples
 can be easily imagined for certain methods that operate on images, it is more
 challenging to figure out what an equivalent transformation would be for a
 patient's clinical test results. Consequently, it may be hard to employ
-adversarial training examples, not to be confused with generative adversarial
-neural networks, with other applications. Finally, approaches that transfer
+adversarial training examples with other applications. Finally, approaches that transfer
```

Comment:
Should we add `adversarial training examples` to our new glossary table?  That would help differentiate if from GANs.

Reply:
That would be nice! Most people working in this field call the training procedure using adversarial examples as "adversarial training". We can add both terms if possible.